app/views/overview/_service-instance-row.html

```diff
@@ -0,0 +1,133 @@
+<div class="list-pf-item" ng-class="{ active: row.expanded }">
+  <div class="list-pf-container" ng-click="row.toggleExpand($event)">
+
+    <div class="list-pf-chevron">
+      <div ng-include src=" 'views/overview/_list-row-chevron.html' " class="list-pf-content"></div>
+    </div>
+
+    <div class="list-pf-content">
+      <div class="list-pf-name">
+        <h3>
+          <a ng-href="{{row.apiObject | navigateResourceURL}}">
```

Comment:
i'm thinking about not making this a link right now, I know it will send you to Other Resources but thats a really crappy experience, and i dont think we even want people doing Edit YAML on service instances right now, and we already have Deprovision from the overview